substrate/frame/broker/src/benchmarking.rs

```diff
@@ -852,6 +852,27 @@ mod benches {
 		}
 	}
 
+	#[benchmark]
+	fn do_tick_base() -> Result<(), BenchmarkError> {
+		setup_and_start_sale::<T>()?;
+
+		advance_to::<T>(5);
+
+		let mut status = Status::<T>::get().ok_or(BenchmarkError::Weightless)?;
```

Comment:
In what case is there no `Status` in storage?  
I just want to ensure that the `Weightless` error is used correctly, since it will return Weight=0.

Reply:
When `do_tick` is called before `start_sales` has been called, there won't be any `Status` in storage.

Reply:
But the benchmark should be the worst case. Otherwise we need one `do_tick_no_sales` and one `do_tick_with_sales`.

Reply:
Got it, will update this to `unwrap` as well.

Reply:
```suggestion
		let mut status = Status::<T>::get().unwrap();
```